### ISSUE TITLE
false positive R0204 (redefinition of type) when using "from __future__ import division"

### ISSUE THREAD

**Issue by hayne** [PyCQA/pylint#781]
``` python
from __future__ import division
x = 1.0
x = 3 / 4
print "x is:", x
```

The above code gives an R0204 warning ("Redefinition of x type from float to int).
This is incorrect since the "from **future** import division" makes the division be floating-point division
(as is also seen from the value (0.75) printed by the last statement).

This is with PyLint 1.5.3 on OS X.

**Comment by PCManticore** [comment]
Unfortunately, redefined-variable-type has some issues, there are multiple issues about it. You can disable the error until we'll fix them. Thank you for the report.

**Comment by PCManticore** [comment]
Closing. ``redefined-variable-type`` is going to be an extension, disabled by default. It needs a lot more attention than we can offer to it.
